File: app/src/main/java/ru/melod1n/vk/api/model/VKPhotoSize.java
package ru.melod1n.vk.api.model;

import org.json.JSONObject;

public class VKPhotoSize {

    private String type;
    private String url;
    private int height;
    private int width;

    public VKPhotoSize(JSONObject o) {
        setType(o.optString("type"));
        setUrl(o.optString("url"));
        setHeight(o.optInt("height"));
        setWidth(o.optInt("width"));
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public int getHeight() {
        return height;
    }

    public void setHeight(int height) {
        this.height = height;
    }

    public int getWidth() {
        return width;
    }

    public void setWidth(int width) {
        this.width = width;
    }

}

File: app/src/main/java/ru/melod1n/vk/api/model/VKPhotoSize.kt
package ru.melod1n.vk.api.model

import org.json.JSONObject
import java.io.Serializable

class VKPhotoSize(o: JSONObject) : Serializable {
    var type: String? = null
    var url: String? = null
    var height = 0
    var width = 0

    companion object {
        private const val serialVersionUID = 1L
    }

    init {
        type = o.optString("type")
        url = o.optString("url")
        height = o.optInt("height")
        width = o.optInt("width")
    }
}